random-beans/src/main/java/io/github/benas/randombeans/util/ReflectionUtils.java

```diff
@@ -265,4 +267,45 @@ public static boolean isWildcardType(final Type type) {
         return actualTypeArguments;
     }
 
+    public static <T> Randomizer<T> newInstance(Class<T> clazz, String[] args) throws IllegalAccessException, InvocationTargetException, InstantiationException {
+        if(args != null && args.length > 0) {
+            for(Constructor c : clazz.getConstructors()) {
+                if(c.getParameterCount() > 0 && c.getParameterCount() == args.length) {
+                    Object[] nArgs = new Object[args.length];
+                    Class[] argTypes = c.getParameterTypes();
+                    for(int x=0; x < args.length; x++) {
+                        Class<?> argType = argTypes[x];
+                        String a = args[x];
+                        nArgs[x] = castPrimitive(argType, a);
```

Comment:
Do we really need to cast to primitive types? I think the method name should be just `cast` since it returns wrapper types. What do you think?